vmm/include/modules/security_info.h

```diff
@@ -28,4 +28,7 @@ void *mov_secinfo_to_internal(void *src);
   Then the allocated memory will be freed. All parameters can't be NULL. Returns number of byte moved */
 uint32_t mov_secinfo_from_internal(void *dest, void *handle);
 
+/* Copy security info from eVmm memory to dest */
+uint32_t copy_secinfo_from_internal(void *dest, void *src);
```

Comment:
no need. we should try to make sure there's only 1 copy of sec_info at any time.
I don't see a reason to do a "copy" instead of "move"